Modules/Devices/src/Devices.Application/PushNotifications/Commands/DeleteDeviceRegistration/DeleteDeviceRegistrationCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+
+namespace Backbone.Modules.Devices.Application.PushNotifications.Commands.DeleteDeviceRegistration;
+
+public class DeleteDeviceRegistrationCommand : IRequest<Unit>
+{
+    public string Platform { get; set; }
+    public string Handle { get; set; }
+    public string AppId { get; set; }
```

Comment:
I explicitly said "The delete command won't have any properties, because they don't make any sense"